doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -369,6 +369,12 @@ described elsewhere
 
          SearchRewrite (_ + _ + _).
 
+.. flag:: Search Blacklist Locals
+
+   By default :cmd:`Search` excludes lemmas declared with :attr:`local` from its results
+   (except for those from the current module or its parents).
```

Comment:
Is it children instead of parents?

Reply:
It's parents. ie 
~~~coq
Module M.
Local Definition foo := ...
Module N.
Search ...
~~~
foo is not excluded because M is parent of the current module M.N.